Source/Core/CorridorGeometry.js

```diff
@@ -597,6 +597,7 @@ define([
             length = topNormals.length;
 
             var extrudeNormals = new Float32Array(length * 6);
+            var i;
```

Comment:
This should be outside the if, since it's used in multiple places outside of this block scope (it would pop up as an error once we have the block scope rule in).  Declare it above `var size` above.